3_javascript/javascript2/week3/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://cdn.jsdelivr.net/npm/confetti-js@0.0.13/dist/index.min.js"></script>
+    <script src="./script.js"></script>
+</head>
+<body>
+    <h2>JavaScript2 - Week 3 HomeWork</h2>
+    <button id="btn-1">Click me!</button>
+    <button id="btn-2">Log location</button>
+    <p id="locationOutput"></p>
+
+    <h3>Fastest Presser</h3>
+    <input  id="durationInput"  type="number"> <br/>   
+
+    <button id="btn-3">start game</button>
```

Comment:
consider using more meaningfull id names like "clickMeButton", "locationButton" etc for better readibility.